### PR TITLE
Initialize darts exercise stub

### PR DESCRIPTION
**Pull request by 0xE282B0**
The stub file was empty, I think we should always provide the input and result fields.

### CODE REVIEW

**Review by KTSnowy — APPROVED** [review]

**Comment by KTSnowy** [comment]
Looks good to me.

File: exercises/practice/darts/src/darts.cob [hunk 1]
```diff
@@ -0,0 +1,10 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. LEAP.
```
Comment (KTSnowy) [inline exercises/practice/darts/src/darts.cob:2]: Actually there's an issue there, it says LEAP instead of darts

Comment (KTSnowy) [reply exercises/practice/darts/src/darts.cob:2]: @0xE282B0 could you push another PR to fix this?

Comment (0xE282B0) [reply exercises/practice/darts/src/darts.cob:2]: Sure, sorry.

**Pull request merged**